Cargo.toml

```diff
@@ -9,3 +9,7 @@ repository = "https://github.com/rust-vmm/vm-device"
 license = "Apache-2.0 OR BSD-3-Clause"
 
 [dependencies]
+vmm-sys-util = { version = "0.8.0" }
```

Comment:
Unless more parameters are used, `vmm-sys-util = "0.8.0"` is simpler. The same goes for the `libc` dev dependency below.

Reply:
Removed the dependency since EventFdTrigger was removed.